owncloudApp/src/main/res/layout/share_public_dialog.xml

```diff
@@ -112,6 +112,34 @@
                 android:inputType="textPassword"
                 android:visibility="gone" />
 
+            <LinearLayout
+                android:id="@+id/layoutPasswordGeneratorButtons"
```

Comment:
I noticed that in this `xml` we don't put the word `layout` in the `id` when we are referring to a layout. Anyway, looking a bit at other layouts in most cases we are putting it at the end and not at the beginning. Just an observation.